docs/reference/setup/install/docker.asciidoc

```diff
@@ -49,7 +49,7 @@ Elasticsearch can be quickly started for development or testing use with the fol
 
 ["source","sh",subs="attributes"]
 --------------------------------------------
-docker run -p 9200:9200 -e "http.host=0.0.0.0" -e "transport.host=127.0.0.1" {docker-image}
+docker run -p 9200:9200 -e "discovery.type=single-node" {docker-image}
```

Comment:
I think that the transport port should be published too.

Reply:
@jasontedor Thanks for the review! This setting would be used only in the [dev mode section](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run) of the elasticsearch docker documentation.

Since using this discovery.type doesn't allow clusters, what is the importance of setting the `transport.tcp.port` or `transport.publish_port`?

Reply:
A use-case is to test the usage of the transport client.

Reply:
Ah I see you mean adding `-p 9300:9300` right?

Reply:
Yes.

Reply:
@jasontedor Addressed in https://github.com/elastic/elasticsearch/pull/26289/commits/42f21a8421db3a4d80679b93ffc628b469fceddb